src/GraphQL.Tests/Types/AutoRegisteringInputObjectGraphTypeTests.cs

```diff
@@ -232,6 +256,26 @@ private class FieldTests
             public string? Field10 { get; set; }
             [Ignore]
             public string? Field11 { get; set; }
+            public int NotNullIntField { get; set; }
+            public int? NullableIntField { get; set; }
+            public string NotNullStringField { get; set; } = null!;
+            public string? NullableSringField { get; set; }
+            public List<string?> NotNullListNullableStringField { get; set; } = null!;
+            public List<string> NotNullListNotNullStringField { get; set; } = null!;
+            public List<string?>? NullableListNullableStringField { get; set; }
+            public List<string>? NullableListNotNullStringField { get; set; }
+            public IEnumerable<int?> NotNullEnumerableNullableIntField { get; set; } = null!;
+            public IEnumerable<int> NotNullEnumerableNotNullIntField { get; set; } = null!;
+            public IEnumerable<int?>? NullableEnumerableNullableIntField { get; set; }
+            public IEnumerable<int>? NullableEnumerableNotNullIntField { get; set; }
+            public Tuple<int, string>?[] NotNullArrayNullableTupleField { get; set; } = null!;
+            public Tuple<int, string>[] NotNullArrayNotNullTupleField { get; set; } = null!;
+            public Tuple<int, string>?[]? NullableArrayNullableTupleField { get; set; }
+            public Tuple<int, string>[]? NullableArrayNotNullTupleField { get; set; }
+            [Id]
```

Comment:
[Id]
            public int IdField { get; set; }
            public IEnumerable EnumerableField { get; set; } = null!;
            public ICollection CollectionField { get; set; } = null!;

Would like to see tests for
            [Id]
            public int? IdField { get; set; }
            public IEnumerable? EnumerableField { get; set; } = null!;
            public ICollection? CollectionField { get; set; } = null!;

Reply:
the same for `AutoRegisteringObjectGraphTypeTests`

Reply:
Done